maven/spec/dependabot/maven/update_checker_spec.rb

```diff
@@ -59,15 +65,16 @@
     "https://repo.maven.apache.org/maven2/"\
     "com/google/guava/guava/23.6-jre/"
```

Comment:
I think these version URLs need to change to include the file we added, e.g. `guava-23.6-jre.jar`